encoding/api-basics.html

```diff
@@ -17,8 +17,15 @@
 
 function testDecodeSample(encoding, string, bytes) {
   test(function() {
-    assert_equals(new TextDecoder(encoding).decode(new Uint8Array(bytes)), string);
-    assert_equals(new TextDecoder(encoding).decode(new Uint8Array(bytes).buffer), string);
+    const typedArray = new Uint8Array(bytes);
+    assert_equals(new TextDecoder(encoding).decode(typedArray), string);
+    assert_equals(new TextDecoder(encoding).decode(typedArray.buffer), string);
+    if (typeof SharedArrayBuffer === 'function') {
```

Comment:
Put the SharedArrayBuffer stuff in its own test() wrapper and I think it'd be fine. No need to make it conditional therein. (Note that a910ad1791d138e10558c3ab0db1a29c6a8c5058 landed a large number of tests already.)

Reply:
maybe we don't need this PR then, it looks like https://github.com/nodejs/node/tree/master/test/fixtures/wpt being updated would suffice